runtime/rasdump/TextFileStream.cpp

```diff
@@ -25,9 +25,11 @@
 #include "TextFileStream.hpp"
 #include "j9.h"
 #include "j9port.h"
+#include "rasdump_internal.h"
 
 /* Constructor */
 TextFileStream::TextFileStream(J9PortLibrary* portLibrary) :
+	_IsOpen(false),
```

Comment:
Please also initialize `_Buffer`.

Reply:
Isn't `_Buffer` initialized on line 39?

`_Buffer = (char *) OMRPORT_FROM_J9PORT(_PortLibrary)->mem_allocate_memory(OMRPORT_FROM_J9PORT(_PortLibrary), _BufferSize, "TextFileStream::TextFileStream", OMRMEM_CATEGORY_VM);`

Reply:
Ideally all fields should be constructed before the body of the constructor begins; adding `_Buffer(NULL),` before `_IsOpen(false),` does that.